Slim/Player/TranscodingHelper.pm

```diff
@@ -361,8 +361,11 @@ sub getConvertCommand2 {
 	}
 
 	if ($prefs->get('prioritizeNative')) {
-		my ($format) = grep /$type/, @supportedformats;
-		@supportedformats = ($format, grep { $_ !~ $type } @supportedformats) if $format;
+		my @types = $type eq 'wav' ? ('pcm', $type) : ($type);
```

Comment:
You put `pcm` first because with the following re-ordering the order would be reversed again?

Reply:
yes, I want the native $type to be first (indeed, thank for having corrected me on that one) so it has to be last one in the loop